### PR TITLE
feat: added Cover Image and Best Sellers on Home Page

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:
#Description: Added Cover Image and Best Sellers on Home Page. Also, made my site responsive.

#More details:

1. Home Page -
    a.   Added Cover image.
    b.   Added Best Sellers products.
    c.   Added links to Category and Best Sellers. On clicking them, user is directed to Product Listing page.
2. Wishlist and Cart Pages -
    a.   Added gif which indicates Wishlist and Cart is empty.
3. Navbar - 
    a.   Added logo
4. Improved responsiveness.

Screenshots:
coverImg
bestS
gif

### CODE REVIEW

**Comment by sashaboi** [comment]
On my screen the home page image seems to move around a bit, can be addressed while working on the responsiveness for bigger screen. Onkar Deshpande
image

**Review by sashaboi — APPROVED** [review]
Looks great , hope you take care of the responsiveness bit
Onkar

File: src/Components/BestSeller/BestSeller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from "react";
+import { ProductContext } from "../../Context/product-context";
+
+export default function BestSeller() {
+  const { products } = useContext(ProductContext);
+  return (
+    <div className="products-container">
+      {products.map((product) => {
+        return (
```
Comment (arulk8) [inline src/Components/BestSeller/BestSeller.js:9]: I would suggest to use object destructing `.map(({id, title, price})=>)`

File: src/Components/CartCard/CartCard.js [hunk 2]
```diff
@@ -47,7 +49,15 @@ export default function CartCard({ cartList }) {
                   >
                     Remove from Cart
                   </button>
-                  <button className="horizontal-btn moveItem-btn">
+                  <button
+                    className="horizontal-btn moveItem-btn"
+                    onClick={() =>
+                      wishlistDispatch({
+                        type: "ADD_TO_WISHLIST",
```
Comment (arulk8) [inline src/Components/CartCard/CartCard.js:56]: this type values can be moved to a separate file and it can be referred in any number of places

Comment (arulk8) [reply src/Components/CartCard/CartCard.js:56]: the shop now button is showing a box below it.

**Pull request merged**